geode/src/test/java/org/apache/calcite/adapter/geode/rel/RelationalJdbcExample.java

```diff
@@ -94,10 +81,81 @@ public static void main(String[] args) throws Exception {
       LOGGER.info("Result entry: " + buf.toString());
       buf.setLength(0);
     }
+
+    GeodeUtils.closeClientCache();
     resultSet.close();
     statement.close();
     connection.close();
   }
+
+  private static void runExampleWithLocatorHostPort() throws Exception {
+    final String geodeModelJson =
+        "inline:"
+            + "{\n"
+            + "  version: '1.0',\n"
+            + "  schemas: [\n"
+            + "     {\n"
+            + "       type: 'custom',\n"
+            + "       name: 'TEST',\n"
+            + "       factory: 'org.apache.calcite.adapter.geode.rel.GeodeSchemaFactory',\n"
+            + "       operand: {\n"
+            + "         locatorHost: 'localhost', \n"
+            + "         locatorPort: '10334', \n"
+            + "         regions: 'BookMaster,BookCustomer,BookInventory,BookOrder', \n"
+            + "         pdxSerializablePackagePath: 'org.apache.calcite.adapter.geode.domain.*' \n"
+            + "       }\n"
+            + "     }\n"
+            + "   ]\n"
+            + "}";
+
+    runExampleForGivenGeodeModelJson(geodeModelJson);
+  }
+
+  private static void runExampleWithJndiClientCacheObject() throws Exception {
+    Hashtable env = new Hashtable();
+    env.put(Context.INITIAL_CONTEXT_FACTORY, GeodeSchemaFactory.GEODE_JNDI_INITIAL_CONTEXT_FACTORY);
+    Context context = new InitialContext(env);
+    GemFireCache gemFireCache = GeodeUtils.createClientCache("localhost", 10334,
+        "org.apache.calcite.adapter.geode.domain.*", true);
+    context.bind("java:/TestClientCacheObject", gemFireCache);
+    final String geodeModelJson =
+        "inline:"
+            + "{\n"
+            + "  version: '1.0',\n"
+            + "  schemas: [\n"
+            + "     {\n"
+            + "       type: 'custom',\n"
+            + "       name: 'TEST',\n"
+            + "       factory: 'org.apache.calcite.adapter.geode.rel.GeodeSchemaFactory',\n"
+            + "       operand: {\n"
+            + "         geodeClientCacheObjectKey: 'java:/TestClientCacheObject', \n"
```

Comment:
If we agree that SchemaFactories should allow retrieval of objects using JNDI (not yet decided) then they should have consistent name for Elastic, Geode, Mongo :  `jndi-name` or `jndiName`  etc.

Reply:
Will stick to camel case naming convention **testClientCacheObjectKey**